src/core_plugins/vega/public/data_model/es_query_parser.js

```diff
@@ -56,20 +57,39 @@ export class EsQueryParser {
     if (body === undefined) {
       url.body = body = {};
     } else if (!_.isPlainObject(body)) {
-      throw new Error('url.body must be an object');
+      throw new Error(i18n.translate('vega.esQueryParser.urlBodyValueTypeErrorMessage', {
+        defaultMessage: '{configName} must be an object',
+        values: { configName: 'url.body' },
+      }));
     }
 
     // Migrate legacy %context_query% into context & timefield values
     const legacyContext = url[LEGACY_CONTEXT];
     delete url[LEGACY_CONTEXT];
     if (legacyContext !== undefined) {
       if (body.query !== undefined) {
-        throw new Error(`Data url must not have legacy "${LEGACY_CONTEXT}" and "body.query" values at the same time`);
+        throw new Error(i18n.translate('vega.esQueryParser.dataUrlMustNotHaveLegacyAndBodyQueryValuesAtTheSameTimeErrorMessage', {
+          defaultMessage: 'Data url must not have legacy "{legacyContext}" and "{bodyQueryConfigName}" values at the same time',
```

Comment:
This one is a bit tricky - `url` is also a param here - it's a field that exists inside the `data` section.  I'm worried that this meaning might get lost in translation.

Reply:
Put 'url' to values' Is it okay or should I put 'Data url' instead?